docs/installation/server-config/security.rst

```diff
@@ -4,20 +4,100 @@ Securing KSQL
+        producer.confluent.monitoring.interceptor.ssl.truststore.password=confluent
+        producer.confluent.monitoring.interceptor.ssl.keystore.location=/etc/kafka/secrets/kafka.client.keystore.jks
+        producer.confluent.monitoring.interceptor.ssl.keystore.password=confluent
+        producer.confluent.monitoring.interceptor.ssl.key.password=confluent
+        producer.confluent.monitoring.interceptor.security.protocol=SASL_SSL
+        producer.confluent.monitoring.interceptor.sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username="client" password="client-secret";
+        producer.confluent.monitoring.interceptor.sasl.mechanism=PLAIN
+
+        # Consumer Confluent Monitoring Interceptors for Control Center streams monitoring
+        consumer.interceptor.classes=io.confluent.monitoring.clients.interceptor.MonitoringConsumerInterceptor
+        consumer.confluent.monitoring.interceptor.ssl.truststore.location=/etc/kafka/secrets/kafka.client.truststore.jks
+        consumer.confluent.monitoring.interceptor.ssl.truststore.password=confluent
+        consumer.confluent.monitoring.interceptor.ssl.keystore.location=/etc/kafka/secrets/kafka.client.keystore.jks
+        consumer.confluent.monitoring.interceptor.ssl.keystore.password=confluent
+        consumer.confluent.monitoring.interceptor.ssl.key.password=confluent
+        consumer.confluent.monitoring.interceptor.security.protocol=SASL_SSL
+        consumer.confluent.monitoring.interceptor.sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username="client" password="client-secret";
+        consumer.confluent.monitoring.interceptor.sasl.mechanism=PLAIN
+
+    The exact settings you need depend on what SASL mechanism your Kafka cluster is using and how your SSL certificates
+    are signed. For full details, see the `Security section of the Kafka documentation <http://kafka.apache.org/documentation.html#security>`__.
+
+#.  Start KSQL server with your configuration file specified.
+
+    .. code:: bash
+
+        $ <path-to-confluent>/bin/ksql-server-start <path-to-confluent>/etc/ksql/ksql-server.properties
+
+    .. tip:: After KSQL is started, you can view your settings with the ``SHOW PROPERTIES;`` KSQL command.
+
+---------------------------------------------------
+Using KSQL with a secured Confluent Schema Registry
+---------------------------------------------------
+
+If you are using KSQL with HTTPS to Confluent Schema Registry, you must set the ``ksql.schema.registry.url`` to HTTPS
+(as shown in the example configuration) and set the ``KSQL_OPTS`` environment variable to define the credentials to use
+when communicating with the Confluent Schema Registry:
+
+.. code:: bash
+
+    # Define KSQL security credentials when communicating with the Confluent Schema Registry via HTTPS
+    $ export KSQL_OPTS="-Djavax.net.ssl.trustStore=/etc/kafka/secrets/kafka.client.truststore.jks
```

Comment:
It should be clear when to run this command.  For example, before or after you start the KSQL server? (I suppose you must run this command before starting the KSQL server, and in the same terminal, otherwise the KSQL server will not pick it up.)

Reply:
added `This must be run before starting the KSQL server.`

Reply:
I would just add the start command afterwards, e.g.

```
$ export KSQL_OPTS=blah
$ ./ksql-server-start <blah>/etc/ksql/ksql-server.properties
or
$ ./confluent start
```

Using this with `confluent start` will also need testing to double check it actually works as expected.